DataMigration/src/test/java/org/sagebionetoworks/dian/datamigration/SynapseTests.java

```diff
@@ -0,0 +1,307 @@
+        assertNotNull(header);
+        assertEquals("test_session_schedule", headerTestSessionSchedule.getName());
+
+        EntityHeader headerTestWakeSleep = subFolderResponse.getPage().get(2);
+        assertNotNull(header);
+        assertEquals("wake_sleep_schedule", headerTestWakeSleep.getName());
+    }
+
+    @Test
+    public void testDownload_testSession_ZIP() throws SynapseException, IOException {
+        EntityChildrenRequest request = new EntityChildrenRequest();
+        request.setParentId(project.getId());
+        request.setIncludeTypes(Lists.newArrayList(EntityType.folder));
+        EntityChildrenResponse response = synapse.getEntityChildren(request);
+
+        // NavigateFolders
+        EntityHeader header = response.getPage().get(0);
+        EntityChildrenRequest subFolderRequest = new EntityChildrenRequest();
+        subFolderRequest.setParentId(header.getId());
+        subFolderRequest.setIncludeTypes(Lists.newArrayList(EntityType.folder));
+        EntityChildrenResponse subFolderResponse = synapse.getEntityChildren(subFolderRequest);
+        EntityHeader headerTestSession = subFolderResponse.getPage().get(0);
+
+        // Download ZIPs
+        EntityChildrenRequest testSessionZipRequest = new EntityChildrenRequest();
+        testSessionZipRequest.setParentId(headerTestSession.getId());
+        testSessionZipRequest.setIncludeTypes(Lists.newArrayList(EntityType.file));
+
+        EntityChildrenResponse testSessionZipResponse = synapse.getEntityChildren(testSessionZipRequest);
+        assertNotNull(testSessionZipResponse);
+        assertNotNull(testSessionZipResponse.getPage());
+        assertEquals(1, testSessionZipResponse.getPage().size());
+
+        String zipId = testSessionZipResponse.getPage().get(0).getId();
+        String zipFilename = testSessionZipResponse.getPage().get(0).getName();
+
+        FileEntity fileEntity = synapse.getEntity(zipId, FileEntity.class);
+        String fileHandleId = fileEntity.getDataFileHandleId();
+        FileHandleAssociation fileHandleAssociation = new FileHandleAssociation();
+        fileHandleAssociation.setFileHandleId(fileHandleId);
+        fileHandleAssociation.setAssociateObjectId(zipId);
+        fileHandleAssociation.setAssociateObjectType(FileHandleAssociateType.FileEntity);
+
+        File testSessionsZipFile = File.createTempFile("TestSessions.zip", null);
+        testSessionsZipFile.deleteOnExit();
+        synapse.downloadFile(fileHandleAssociation, testSessionsZipFile);
+
+        String testSessionsChecksumActual = MD5ChecksumHelper.getMD5Checksum(testSessionsZipFile);
+        // Expected was calculated by downloading the zip separately in a browser and getting the checksum
+        assertEquals("cda0a862c9e90dacbe0ba33c4caecaaf", testSessionsChecksumActual);
+    }
+
+    @Test
+    public void testDownload_SessionSchedule_Zip() throws SynapseException, IOException {
```

Comment:
Not clear what testDownload_SessionSchedule_Zip is testing. There's no reference to the logic in SynapseUtil and this code need not test that the Synapse client works.